sklearn/model_selection/tests/test_validation.py

```diff
@@ -134,38 +135,38 @@ def test_cross_val_score():
     for a in range(-10, 10):
         clf.a = a
         # Smoke test
-        scores = cross_val_score(clf, X, y)
```

Comment:
Is this comment related to your [previous comment](https://github.com/scikit-learn/scikit-learn/pull/5703/files#r49243981)?